conda/recipes/pynvjitlink/recipe.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  head_rev: ${{ git.head_rev(".")[:8] }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  version: ${{ load_from_file("pynvjitlink/VERSION") | trim }}
+
+package:
+  name: pynvjitlink
+  version: 0.5.2
+source:
+  path: ../../..
+
+build:
+  # prefix_detection:
+  #   # See https://github.com/rapidsai/build-planning/issues/160
+  #   # Blanket ignore here as there are quite a few shared objects shipped in cuml
+  #   ignore_binary_files: True
+  string: py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+  script:
+    content: |
+      python -m pip install . -vv
+    secrets:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+    env:
+      CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
+      CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
+      CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
+      CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
+      SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
+      SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
+      SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
+      SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
+      SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
+      SCCACHE_S3_KEY_PREFIX: pynvjitlink/${{ env.get("RAPIDS_CONDA_ARCH") }}/cuda${{ cuda_major }}
+
+requirements:
+  build:
+    - cmake >=3.24.4,!=3.30.0
+    - ninja
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("cuda") }}
+    - ${{ stdlib("c") }}
```

Comment:
Personally find it a little easier to have compilers first and then build dependencies. That way it is easy to see what language features are used. Then look at build tools after